platform/source/lib/ax/platform/rest.py

```diff
@@ -202,21 +203,19 @@ def put_spot_instance_config():
     payload = {'enabled': enabled_str}
 
     # Get "asgs" option
-    (asgs,) = _get_optional_arguments('asgs')
-    if asgs is not None:
-        assert asgs in SpotInstanceOption.VALID_SPOT_INSTANCE_OPTIONS, "Unknown spot-instance-option " + asgs
-        asg_option = ""
-        if asgs == SpotInstanceOption.PARTIAL_SPOT:
-            asg_option = asg_manager.get_variable_asg()["AutoScalingGroupName"]
-        elif asgs == SpotInstanceOption.ALL_SPOT:
-            for asg in asg_manager.get_all_asgs():
-                asg_option = asg_option + asg["AutoScalingGroupName"] + " "
-            # Remove the last <space>
-            asg_option = asg_option[:-1]
-        elif asgs == SpotInstanceOption.NO_SPOT:
+    (option,) = _get_optional_arguments('spot_instances_option')
```

Comment:
Please document what an input data look like so people in the community would know how to use these APIs

Reply:
I'll add a README in the next commit. Will update the associated UI bug (https://github.com/argoproj/argo/issues/171) as soon as this is ready to checkin.